prusti-interface/src/lifetime_formatter.rs

```diff
@@ -0,0 +1,58 @@
+use rustc_middle::ty;
+
```

Comment:
I think it would be cleaner to create a module `prusti-interface/src/lifetimes/mod.rs` and move `prusti-interface/src/lifetime_formatter.rs` and `prusti-interface/src/environment/mir_dump/lifetimes.rs` into it.

Reply:
done, I had to add a lot of `pub(crate)`s